src/store/index.ts

```diff
@@ -23,22 +23,38 @@ import analytics from 'store/cache/analytics/slice'
 import music from 'store/cache/music/slice'
 import account from 'store/account/slice'
 import pageHistory from 'store/pageHistory/slice'
+import api from 'store/api/slice'
 
 declare global {
   interface Window {
     aud: any
     store: any
     client: ApolloClient<any>
+    hostedClient: ApolloClient<any>
   }
 }
 
 const gqlUri = process.env.REACT_APP_GQL_URI
+const gqlBackupUri = process.env.REACT_APP_GQL_BACKUP_URI
 
 export const client = new ApolloClient({
   uri: gqlUri,
   cache: new InMemoryCache()
 })
 
+let hostedClient = null
+if (gqlBackupUri) {
+  hostedClient = new ApolloClient({
```

Comment:
is instantiating a second apollo expensive? should we maybe consider doing this on demand?

Reply:
I do not believe it is expensive - looking through their code it just instantiates a class instance, but it's probably better practice to create on demand - can do